docs/index.rst

```diff
@@ -29,6 +29,7 @@ If you're trying to **use** coala, you should have a look at our
    :hidden:
 
    Newcomers' Guide <Developers/Newcomers_Guide>
+   Documentation coala settings <Developers/Doc_coala_Settings>
```

Comment:
also I dislike the filename, maybe just coala_settings.rst and the title could be `coala Settings`